chips/opentitan-earlgrey/chip.toml

```diff
@@ -0,0 +1,32 @@
+# These address are taken from the qemu opentitan machine
+# https://github.com/qemu/qemu/blob/master/hw/riscv/opentitan.c#L32
+
+[gpio0]
+address = 0x4004_0000
+size = 0x1000
+
+[uart0]
+address = 0x4000_0000
+size = 0x1000
+
+[aontimer]
+address = 0x4047_0000
+size = 0x1000
+interrupts = { bark = 158 }
```

Comment:
Why is this named differently (`bark`) than other interrupts that use `irq`? I understand that the aontimer spec refers to this threshold as bark, but it eventually triggers an interrupt. Do we want a different name here?

---

Also, this `bark` element doesn't really get used anywhere. I would expect something in app.toml to use aontimer.bark and support in build/xtask to check this `bark` key. But, neither exists. So, wondering if this is just a leftover?

Reply:
I don't know if `irq = ...` is a convention, if it is a convention it should've been `irq = 158` instead. It used to be used in the opentitan task toml, but there might have been changes that make this no longer necessary. @beezow

Reply:
Looking at [how xtask handles this](https://github.com/rivosinc/hubris/blob/2563c2c4550c0f0a558364aea200ddacc227ca2a/build/xtask/src/dist.rs#L1725), it seems to support any name for the interrupt. So, basically app.toml should be able to use something like `aontimer.bark`. Given that, I think it should be okay to use `bark` here.